docs/vhost-user-net-testing.md

```diff
@@ -0,0 +1,126 @@
+# How to test Vhost-user net with OpenVSwitch/DPDK
+
+The purpose of this document is to illustrate how to test vhost-user-net in cloud-hypervisor with OVS/DPDK as the backend.
+
+## Framework
+
+It's a simple test to validate the communication between two virtual machine, connecting them to vhost-user ports respectively provided by `OVS/DPDK`.
+```
+             +----+----------+          +-------------+-----------+-------------+          +----------+----+  
+             |    |          |          |             |           |             |          |          |    | 
+             |    |vhost-user|----------| vhost-user  |   ovs     | vhost-user  |----------|vhost-user|    |           
+             |    |net device|          | port 1      |           | port 2      |          |net device|    |           
+             |    |          |          |             |           |             |          |          |    |           
+             |    +----------+          +-------------+-----------+-------------+          +----------+    |           
+             |               |          |                                       |          |               |           
+             |vm1            |          |                  dpdk                 |          |           vm2 |           
+             |               |          |                                       |          |               |           
+          +--+---------------------------------------------------------------------------------------------+--+        
+          |  |                                       hugepages                                             |  |        
+          |  +---------------------------------------------------------------------------------------------+  |        
+          |                                                                                                   |        
+          |                                              host                                                 |        
+          |                                                                                                   |  
+          +---------------------------------------------------------------------------------------------------+        
+```
+## Prerequisites
+
+Prior to running the test, the following steps need to be performed, [vhost-user-ovs-dpdk](https://wiki.qemu.org/Documentation/vhost-user-ovs-dpdk) is a good reference for detailing them.
+- Enable hugepages
+- Install kernel-devel
+- Build DPDK from source
+- Build OVS from source
+
```

Comment:
It is good to add some detailed steps on how to achieve the above tasks.  Or add some useful links on how to do it. Looks like the document referred is a little bit old. I tried the steps according to the documents and run into the problem.

Reply:
As what we discussed through the mail, the qemu web page has redhat as an example for the installation, so please have a try on redhat and let me know what the problem you have.

Reply:
oh, sorry, I could not find a redhat machine here.  But I can try it on Ubunutu machine basing on this document.  Is it helpful?  :)